ucs/clientlist.cpp

```diff
@@ -1097,7 +1148,8 @@ void Client::JoinChannels(std::string ChannelNameList) {
 	safe_delete(outapp);
 }
 
-void Client::LeaveChannels(std::string ChannelNameList) {
+void Client::LeaveChannels(std::string ChannelNameList, bool commandDirected) {
+	LogDebug("Triggered:LeaveChannels on {},  Command Directed: {}", ChannelNameList, commandDirected);
```

Comment:
This is duplication of the `LogInfo` below it, just consolidate the logging statements into one

Reply:
Removed